packages/jest-runtime/src/script_transformer.js

```diff
@@ -242,12 +242,12 @@ export default class ScriptTransformer {
       }
     }
 
-    if (mapCoverage) {
-      if (!transformed.map) {
-        const inlineSourceMap = convertSourceMap.fromSource(transformed.code);
-        if (inlineSourceMap) {
-          transformed.map = inlineSourceMap.toJSON();
-        }
+    if (!transformed.map) {
```

Comment:
Since `mapCoverage` is removed from this file, there's only a single place in the sourcecode left: https://github.com/facebook/jest/blob/36c705bb1eac380e0926487bbe8b8db0aa2fa2e8/packages/jest-cli/src/reporters/coverage_reporter.js#L86-L88

Does it make sense to keep that?

Reply:
We can remove it. I re-ran the testcases making that change and the coverage is getting properly remapped in two of the integration testcases. 
I manually ran jest inside those testcases, and the html coverage report that was generated seems to match the coverage %age.

* At this point I have to ask, are we okay with removing `mapCoverage` completely. Or should it be there and default to false.

Reply:
I personally think that option should be removed completely.

Reply:
For future reviewers, this comment is resolved